charts/apisix-dashboard/templates/configmap.yaml

```diff
@@ -30,6 +30,7 @@ data:
         port: {{ .listen.port }}
       {{- with .etcd }}
       etcd:
+        prefix: {{ .prefix | quote }}
```

Comment:
Could you also describe it in the README?

Reply:
> Could you also describe it in the README?

@tokers @juzhiyuan 

Maybe you mean something like this [configuration list description](https://github.com/apache/apisix-helm-chart/tree/master/charts/apisix#parameters)?  Maybe we can create another PR to handle this.

Reply:
Yes, but I think it's quite handful to add some descriptions for it, maybe in the same PR is better.

Reply:
Done! 

@juzhiyuan @tokers  PTAL!

Reply:
@Gallardot Thanks!